### PR TITLE
refactor: refactor loading modal

### PR DESCRIPTION
**Pull request by farhan-nurzi-deriv**
## Changes:

Please provide a summary of the change.

### Screenshots:
Screenshot 2023-07-18 at 5 57 05 PM

### CODE REVIEW

**Review by ameerul-deriv — APPROVED** [review]
LGTM

File: packages/p2p/src/components/modal-manager/modals/loading-modal/__tests__/loading-modal.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+    it('should render loading modal', () => {
+        render(<LoadingModal />);
+        expect(screen.getByTestId('dt_initial_loader')).toBeInTheDocument();
```
Comment (ako-deriv) [inline packages/p2p/src/components/modal-manager/modals/loading-modal/__tests__/loading-modal.spec.tsx:25]: hi, if you check this [document](https://testing-library.com/docs/queries/about/#priority) you will see that getByTestId has the lowest priority in the all the selectors. I recommend you to use getByRole or getByLabelText and ... if it's possible.

Comment (farhan-nurzi-deriv) [reply packages/p2p/src/components/modal-manager/modals/loading-modal/__tests__/loading-modal.spec.tsx:25]: Thanks @ali-hosseini-deriv , I will check if i can use other selector 🙏

Comment (farhan-nurzi-deriv) [reply packages/p2p/src/components/modal-manager/modals/loading-modal/__tests__/loading-modal.spec.tsx:25]: @ali-hosseini-deriv  This modal component render `loading.tsx` component which is only a loader. I dont think i can use other selector here.

**Review by nada-deriv — APPROVED** [review]

File: packages/p2p/src/components/modal-manager/modals/loading-modal/__tests__/loading-modal.spec.tsx (hunk 1 above)
Comment (ako-deriv) [reply packages/p2p/src/components/modal-manager/modals/loading-modal/__tests__/loading-modal.spec.tsx:25]: there would be an icon or something similar to that in the DOM right?
maybe you can use that.

Comment (farhan-nurzi-deriv) [reply packages/p2p/src/components/modal-manager/modals/loading-modal/__tests__/loading-modal.spec.tsx:25]: it only contains divs without any text 😢 

image

**Review by niloofar-deriv — APPROVED** [review]

**Review by farrah-deriv — APPROVED** [review]

**Pull request merged**